slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -754,9 +747,14 @@ void ParseAndValidateArguments(const vector<string> &args,
 
     auto map_rate_sec = ArgParser(args, "-map_rate_sec=");
     if (map_rate_sec.empty()) {
-        map_rate_sec = "0";
+        throw runtime_error("No map data rate specified");
```

Comment:
[nit] Mention `map_rate_sec` in the error message. Example: `"map_rate_sec value is required"`